internal/ingress/controller/parser/parser_test.go

```diff
@@ -198,7 +199,7 @@ func TestSecretConfigurationPlugin(t *testing.T) {
 					Namespace: "default",
 					Annotations: map[string]string{
 						"plugins.konghq.com":          "foo-plugin",
```

Comment:
Same as above, for `annotations.deprecatedPluginsKey`

Also for other annotation keys used throughout the codebase. Those should be defined as consts in the annotations package, exported and unversally used.

Reply:
ditto https://github.com/Kong/kubernetes-ingress-controller/pull/815#discussion_r479437069

Reply:
fixed in commit 36e4d07